### PR TITLE
Attention mask fixed. Documentation: @sgugger

### PR DESCRIPTION
**Pull request by kant**
# What does this PR do?
* Attention mask fixed (line 217)
* typo fixed (paragraph 326)

Fixes # (issue)
No previous issue related

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [ ] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review?
@osanseviero 
@Narsil 
@ydshieh 
@sgugger 
@omarespejel 
@patrickvonplaten 

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.

### CODE REVIEW

File: docs/source/es/quicktour.mdx [hunk 1]
```diff
@@ -214,7 +214,7 @@ Pasa tu texto al tokenizador:
 El tokenizador devolverá un diccionario conteniendo:
 
 * [input_ids](./glossary#input-ids): representaciones numéricas de los tokens.
-* [atttention_mask](.glossary#attention-mask): indica cuáles tokens deben ser atendidos.
```
Comment (ydshieh) [inline docs/source/es/quicktour.mdx:217]: Good catch! Could you also fix the other 3 occurrences in other `quicktour.mdx` files 🙏, thank you!

File: docs/source/es/quicktour.mdx [hunk 2]
```diff
@@ -323,7 +323,7 @@ final (como softmax) porque la función de activación final es comúnmente fusi
 
 </Tip>
 
-Los modelos son [`torch.nn.Module`](https://pytorch.org/docs/stable/nn.html#torch.nn.Module) o [`tf.keras.Model`](https://www.tensorflow.org/api_docs/python/tf/keras/Model) estándares así que podrás usarlos en tu training loop usual. Sin embargo, para facilitar las cosas, 🤗 Transformers provee una clase [`Trainer`] para PyTorch que añade funcionalidades para entrenamiento distribuido, precición mixta, y más. Para TensorFlow, puedes usar el método `fit` desde [Keras](https://keras.io/). Consulta el [tutorial de entrenamiento](./training) para más detalles.
+Los modelos son [`torch.nn.Module`](https://pytorch.org/docs/stable/nn.html#torch.nn.Module) o [`tf.keras.Model`](https://www.tensorflow.org/api_docs/python/tf/keras/Model) estándares así que podrás usarlos en tu training loop usual. Sin embargo, para facilitar las cosas, 🤗 Transformers provee una clase [`Trainer`] para PyTorch que añade funcionalidades para entrenamiento distribuido, precisión mixta, y más. Para TensorFlow, puedes usar el método `fit` desde [Keras](https://keras.io/). Consulta el [tutorial de entrenamiento](./training) para más detalles.
```
Comment (ydshieh) [inline docs/source/es/quicktour.mdx:326]: Google translate tells me `precisión` is correct, but I would like @osanseviero to double check :-)

**Review by ydshieh — APPROVED** [review]
Thank you @kant ❤️

**Comment by sgugger** [comment]
Same as your other PRs, there is an issue with your CircleCI permissions, the tests won't run.
Could you try refreshing your permissions as shown [here](https://support.circleci.com/hc/en-us/articles/360048210711-How-to-Refresh-User-Permissions-)?

File: docs/source/es/quicktour.mdx (hunk 2 above)
Comment (osanseviero) [reply docs/source/es/quicktour.mdx:326]: Yes it's correct!
